WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginMagicLinkSentFragment.java

```diff
@@ -0,0 +1,119 @@
+    public static final String TAG = "login_magic_link_sent_fragment_tag";
+
+    private static final String ARG_EMAIL_ADDRESS = "ARG_EMAIL_ADDRESS";
+
+    private LoginListener mLoginListener;
+
+    private String mEmail;
+
+    public static LoginMagicLinkSentFragment newInstance(String email) {
+        LoginMagicLinkSentFragment fragment = new LoginMagicLinkSentFragment();
+        Bundle args = new Bundle();
+        args.putString(ARG_EMAIL_ADDRESS, email);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    public LoginMagicLinkSentFragment() {}
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            mEmail = getArguments().getString(ARG_EMAIL_ADDRESS);
+        }
+
+        setHasOptionsMenu(true);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.login_magic_link_sent_fragment, container, false);
+
+        view.findViewById(R.id.login_open_email_client).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mLoginListener != null) {
+                    mLoginListener.openEmailClient();
+                }
+            }
+        });
+
+        view.findViewById(R.id.login_enter_password).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mLoginListener != null) {
+                    mLoginListener.usePasswordInstead(mEmail);
+                }
+            }
+        });
+
+        return view;
+    }
+
+    @Override
+    public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        Toolbar toolbar = (Toolbar) view.findViewById(R.id.toolbar);
+        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);
+
+        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayShowTitleEnabled(false);
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+    }
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (context instanceof LoginListener) {
+            mLoginListener = (LoginListener) context;
+        } else {
+            throw new RuntimeException(context.toString() + " must implement LoginListener");
+        }
+    }
+
+    @Override
+    public void onDetach() {
+        super.onDetach();
+        mLoginListener = null;
+    }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        inflater.inflate(R.menu.menu_login, menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
When unimplemented like it is now, it looks redundant and noisy but down the road, the idea is that each fragment will provide different functionality when Help is tapped (like different metadata or FAQ page jump) so, it's easier to handle like this. Let's revisit after all the screens/features are done. Sounds OK?